GrandCentralBoard/Widgets/WebsiteAnalytics/View/TableDataSource.swift

```diff
@@ -19,7 +19,7 @@ protocol TableViewModel {
 
 final class TableDataSource<T: ViewConfiguring, U: TableViewModel where U.CellViewModel == T.CellViewModel, U.HeaderViewModel == T.HeaderViewModel> : NSObject, UITableViewDelegate, UITableViewDataSource {
 
-    private var viewModel: U
+    var viewModel: U
```

Comment:
I checked, and you can't use name of protocols if they have `associatedType` in them. Sad but true